src/index.html

```diff
@@ -1,17 +1,145 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="section">
+        <legend>Personal information</legend>
+        <label class="form-field"
+          >Surname:
+          <input type="text"
+                 name="surname" 
+                 autocomplete="off" 
+          >
+        </label>
+        <label class="form-field"
+          >Name:
+          <input type="text"
+                 name="name" 
+                 autocomplete="off" 
+          >
+        </label>
+        <label class="form-field"
+          >How old are You?
+          <input type="number"
+                 name="age" 
+                 value="12" 
+                 min="1" 
+                 max="100" 
+          >
+        </label>
+        <label class="form-field"
+          >Full date of birth
+          <input type="date" 
+                 name="dateOfBirth" 
+          >
+        </label>
+        <label class="form-field"
+          >I accept the term of agreement
+          <input type="checkbox"
+                 name="terms" 
+                 required 
+          >
+        </label>
+      </fieldset>
+      <fieldset class="section">
+        <legend>Registration</legend>
+        <label class="form-field"
+          >E-mail:
+          <input
+            type="email"
+            name="email"
+            required
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="form-field"
+          >Password:
+          <input type="password"
+                 name="password" 
+                 minlength="8" 
+          >
+        </label>
+      </fieldset>
+      <fieldset class="section">
+        <legend>An interesting fact about you</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label
+            >Yes
+            <input type="radio"
+                   name="loveCats" 
+            >
+          </label>
+          <label
+            >No
+            <input type="radio"
+                   name="dontLoveCates" 
+            >
+          </label>
+        </div>
```

Comment:
Only one radio button can be selected in the moment
![image](https://user-images.githubusercontent.com/98764641/230712842-10560a14-d792-4831-85a6-0c0e050cc079.png)